3p/3p.js

```diff
@@ -271,6 +271,7 @@ function validateAllowedFields(data, allowedFields) {
     mode: true,
     consentNotificationId: true,
     container: true,
+    ampSlotIndex: true,
```

Comment:
I understand your apprehension, but this whole lifecycle reporter thing is temporary until we make use of amp analytics.